platform/viewer/cypress/integration/common/OHIFCornerstoneToolbar.spec.js

```diff
@@ -0,0 +1,509 @@
+         cy.get('tbody > :nth-child(1) > :nth-child(1)').click();
+         cy.get('.viewport-container').then(($viewport) =>{
+           cy.wrap($viewport)
+             .should('have.attr', 'style', 'height: 100%; width: 100%;')//checking if the layout is correct
+             .its('length')
+             .should('be.eq', 1);
+         })
+    });
+
+
+    it('check if Invert tool will change the colors of the image in the viewport', () => {
+      //Click on More button
+      cy.get('.expandableToolMenu').click();
+      //Verify if overlay is displayed
+      cy.get('.tooltip-toolbar-overlay')
+        .should('be.visible');
+
+      //Click on Invert button
+      cy.get('.tooltip-inner > :nth-child(6)')
+        .click();
+
+      //check on cornerstone if image was inverted
+      cy.window()
+        .its('cornerstone')
+        .then(($cornerstone) => {
+          expect($cornerstone.getEnabledElements()[0].viewport.invert).to.be.true
+        });
+
+      cy.resetViewport();
+    });
+
+
+    it('check if Rotate tool will change the image orientation in the viewport', () => {
+      //Click on More button
+      cy.get('.expandableToolMenu').click();
+      //Verify if overlay is displayed
+      cy.get('.tooltip-toolbar-overlay')
+        .should('be.visible');
+
+      //Click on Rotate button
+      cy.get('.tooltip-inner > :nth-child(7)')
+        .click();
+
+      //Check on cornerstone if image was rotated
+      cy.window()
+        .its('cornerstone')
+        .then(($cornerstone) => {
+          expect($cornerstone.getEnabledElements()[0].viewport.rotation).to.equal(90)
+        });
+
+      cy.resetViewport();
+    });
+
+
+    it('check if Flip H tool will flip the image horizontally in the viewport', () => {
+      //Click on More button
+      cy.get('.expandableToolMenu').click();
+      //Verify if overlay is displayed
+      cy.get('.tooltip-toolbar-overlay')
+        .should('be.visible');
+
+      //Click on Rotate button
+      cy.get('.tooltip-inner > :nth-child(8)')
+        .click();
+
+      //Check on cornerstone if image was flipped horizontally
+      cy.window()
+        .its('cornerstone')
+        .then(($cornerstone) => {
+          expect($cornerstone.getEnabledElements()[0].viewport.hflip).to.be.true
+        });
+
+      cy.resetViewport();
+    });
+
+
+    it('check if Flip V tool will flip the image vertically in the viewport', () => {
+      //Click on More button
+      cy.get('.expandableToolMenu').click();
+      //Verify if overlay is displayed
+      cy.get('.tooltip-toolbar-overlay')
+        .should('be.visible');
+
+      //Click on Rotate button
+      cy.get('.tooltip-inner > :nth-child(9)')
+        .click();
+
+      //Check on cornerstone if image was flipped vertically
+      cy.window()
+        .its('cornerstone')
+        .then(($cornerstone) => {
+          expect($cornerstone.getEnabledElements()[0].viewport.vflip).to.be.true
+        });
+
+      cy.resetViewport();
+    });
```

Comment:
I think we're okay to remove this test.
We're trying to strike a balance between maintainability and increased confidence that the application works.
This test is difficult, because it has us relying on our window'd `cornerstone`.

We have less brittle tests we can rely on to assert that the `cornerstone` extension was registered successfully.
This is a bit nuanced, and certainly up for debate.

Phrased differently, the end-to-end tests are like smoke detectors.
We want enough to make sure we have advanced notice of fire, but too many and it will become a full time job to replace their batteries.
Let's place them strategically.